### DISCUSSION TITLE
push with options?

### DISCUSSION THREAD

**Discussion by jalanh11640** [gitpython-developers/GitPython#1565] [Q&A] [answered]
Can anyone point me to an example of how to push a branch to a remote and specify options?  Here's a git command line example of what I would like to do via python.

git push -o ci.variable="CI_MODULE=myValue"

Thanks!

**Comment by Byron** [comment, marked as answer]
Passing arguments to `git push` can be done directly [as described here](https://gitpython.readthedocs.io/en/stable/tutorial.html#using-git-directly) - similar mechanisms also work with [the more idiomatic way](https://gitpython.readthedocs.io/en/stable/tutorial.html#handling-remotes) of handling remotes.
